[2025-04-04T23:17:46.272+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-04T23:17:46.307+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.create_postgre_stable scheduled__2025-04-01T00:00:00+00:00 [queued]>
[2025-04-04T23:17:46.341+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.create_postgre_stable scheduled__2025-04-01T00:00:00+00:00 [queued]>
[2025-04-04T23:17:46.342+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-04-04T23:17:46.375+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): create_postgre_stable> on 2025-04-01 00:00:00+00:00
[2025-04-04T23:17:46.389+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1608) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-04T23:17:46.389+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'create_postgre_stable', 'scheduled__2025-04-01T00:00:00+00:00', '--job-id', '862', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpq360z0sx']
[2025-04-04T23:17:46.393+0000] {standard_task_runner.py:72} INFO - Started process 1613 to run task
[2025-04-04T23:17:46.394+0000] {standard_task_runner.py:105} INFO - Job 862: Subtask create_postgre_stable
[2025-04-04T23:17:46.493+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.create_postgre_stable scheduled__2025-04-01T00:00:00+00:00 [running]> on host ba06481f48a9
[2025-04-04T23:17:46.688+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='acilrestu12' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='create_postgre_stable' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-01T00:00:00+00:00'
[2025-04-04T23:17:46.691+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-04T23:17:46.751+0000] {sql.py:278} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS dag_runs (
                dt date,
                dag_id character varying(250),
                run_id character varying(250),
                primary key (dt, dag_id, run_id)
            );
        
[2025-04-04T23:17:46.773+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2025-04-04T23:17:46.796+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2025-04-04T23:17:46.828+0000] {sql.py:544} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS dag_runs (
                dt date,
                dag_id character varying(250),
                run_id character varying(250),
                primary key (dt, dag_id, run_id)
            );
        , parameters: None
[2025-04-04T23:17:46.864+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-04T23:17:46.869+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=create_postgre_stable, run_id=scheduled__2025-04-01T00:00:00+00:00, execution_date=20250401T000000, start_date=20250404T231746, end_date=20250404T231746
[2025-04-04T23:17:46.936+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-04T23:17:46.973+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-04T23:17:46.977+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
